src/app/RegistrationPage/page.js

```diff
@@ -0,0 +1,128 @@
+                    </div>
+                    <div className="xsm:h-[63.33px] sm:h-[95px] md:h-[70px]">
+                        <p className="font-poppins font-normal xsm:text-[10.67px] sm:text-[16px] xsm:leading-[16px] sm:leading-[24px]">Are you a young leader with a vision to create positive change in your community and beyond? Apply now to join a network of motivated peers, develop your leadership skills, and gain valuable experience through our immersive program.</p>
+                    </div>
+                    <div className="xsm:h-[16px] sm:h-[30px] xl:hidden 2xl:h-[5px] 2xl:block"></div>
+                    <div className="xsm:h-[33.33px] sm:h-[50px] flex items-center">
+                        <p className="font-poppins font-semibold xsm:text-[10.67px] sm:text-[16px] xsm:leading-[16px] sm:leading-[24px]">Eligibility for Delegates:</p>
+                    </div>
+                    <div className="font-poppins font-normal xsm:text-[10.67px] sm:text-[16px] xsm:leading-[16px] sm:leading-[24px] pl-[9px]">
+                        <ol className="list-decimal list-inside">
+                            <li className="xsm:mb-[9.5px] sm:mb-[13px]">
+                            The Filipino Youth Leadership program is open for delegates between the ages of 18 to 25 with the intention of learning about valuable leadership skills and nation-building.
+                            </li>
+                            <li className="xsm:mb-[9.5px] sm:mb-[13px]">
+                            The delegate must be a Filipino citizen residing in Mindanao.
+                            </li>
+                            <li className="xsm:mb-[9.5px] sm:mb-[13px]">
+                            Individuals with a passion for driving positive change, fostering innovation, and inspiring others are encouraged to apply.
+                            </li>
+                            <li className="xsm:mb-[9.5px] sm:mb-[13px]">
+                            Eligibility is not restricted by geographic location, nationality, gender, race, or any other discriminatory factor.
+                            </li>
+                            <li className="xsm:mb-[9.5px] sm:mb-[13px]">
+                            Participants should have a basic understanding of leadership principles or a willingness to learn and engage actively in the program.
+                            </li>
+                            <li>
+                            While prior leadership experience is not mandatory, a genuine interest in assuming leadership roles and responsibilities is essential.
+                            </li>
+                        </ol>
+                    </div>
+                    <div className="flex justify-center xsm:mt-[35px] sm:mt-[55px] xl:mt-[50px] 2xl:mt-[55px]">
+                        <MainButton href="">apply as a delegate</MainButton>
```

Comment:
Kindly refer to pinned message in GC and add the actual link to the forms created by the FYLP